website/blog/2024-08-29-java-17-and-21-upgrade.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Java Upgrade: 17 or 21?"
+author: "Krishna Rao Chintu"
+---
+---
+
+As Java 11 is becoming obsolete by October 2024, needing to upgrade Java version in our platform and in the generated applications.
```

Comment:
As Java 11 LTS is becoming obsolete by October 2024, needing to upgrade Java version in our platform and in the generated applications.
So need to support to the next LTS versions, Java 17 and Java 21.